tests/Doctrine/Tests/DBAL/Schema/ForeignKeyConstraintTest.php

```diff
@@ -16,14 +16,9 @@ class ForeignKeyConstraintTest extends TestCase
      */
     public function testIntersectsIndexColumns(array $indexColumns, $expectedResult)
     {
-        $foreignKey = new ForeignKeyConstraint(['foo', 'bar'], 'foreign_table', ['fk_foo', 'fk_bar']);
+        $foreignKey = new ForeignKeyConstraint(['foo', '`bar`'], 'foreign_table', ['fk_foo', 'fk_bar']);
```

Comment:
Please design a new test: don't modify existing ones.

If an existing test is being modified out of necessity, we will likely hit a BC break scenario.